projects/1.18/assets/farmers-respite/farmersrespite/lang/zh_cn.json

```
@@ -0,0 +1,72 @@
+{
+  "block.farmersrespite.kettle": "水壶",
+  "block.farmersrespite.rose_hip_pie": "玫瑰果派",
+  "block.farmersrespite.coffee_cake": "咖啡蛋糕",
+  "block.farmersrespite.small_tea_bush": "茶籽",
+  "block.farmersrespite.coffee_bush": "咖啡豆",
+  "block.farmersrespite.wild_tea_bush": "野生茶树",
+
+  "item.farmersrespite.green_tea_leaves": "绿茶叶",
+  "item.farmersrespite.yellow_tea_leaves": "黄茶叶",
+  "item.farmersrespite.black_tea_leaves": "红茶叶",
+  "item.farmersrespite.rose_hips": "玫瑰果",
+  "item.farmersrespite.tea_seeds": "茶籽",
+  "item.farmersrespite.green_tea": "绿茶",
+  "item.farmersrespite.yellow_tea": "黄茶",
+  "item.farmersrespite.black_tea": "红茶",
+  "item.farmersrespite.rose_hip_tea": "玫瑰果茶",
+  "item.farmersrespite.dandelion_tea": "蒲公英茶",
+  "item.farmersrespite.purulent_tea": "脓眼茶",
+  "item.farmersrespite.rose_hip_pie_slice": "玫瑰果派",
+  "item.farmersrespite.green_tea_cookie": "绿茶饼干",
+  "item.farmersrespite.nether_wart_sourdough": "下界疣发酵面包",
+  "item.farmersrespite.black_cod": "红茶炖鳕鱼",
+  "item.farmersrespite.tea_curry": "茶咖喱",
+  "item.farmersrespite.coffee_berries": "咖啡浆果",
+  "item.farmersrespite.coffee": "咖啡",
+  "item.farmersrespite.coffee_cake_slice": "咖啡蛋糕",
+  "item.farmersrespite.blazing_chili": "烈焰辣椒酱",
+
+  "item.farmersrespite.long_green_tea": "绿茶",
+  "item.farmersrespite.long_yellow_tea": "黄茶",
+  "item.farmersrespite.long_black_tea": "红茶",
+  "item.farmersrespite.long_dandelion_tea": "蒲公英茶",
+  "item.farmersrespite.long_coffee": "咖啡",
+  "item.farmersrespite.long_apple_cider": "苹果醋",
+
+  "item.farmersrespite.strong_green_tea": "绿茶",
+  "item.farmersrespite.strong_yellow_tea": "黄茶",
+  "item.farmersrespite.strong_black_tea": "红茶",
+  "item.farmersrespite.strong_purulent_tea": "脓茶",
+  "item.farmersrespite.strong_rose_hip_tea": "玫瑰果茶",
+  "item.farmersrespite.strong_coffee": "咖啡",
+  "item.farmersrespite.strong_apple_cider": "苹果酒",
+  "item.farmersrespite.strong_melon_juice": "西瓜汁",
+
+  "farmersdelight.tooltip.purulent_tea": "延长一个效果15秒",
+  "farmersdelight.tooltip.strong_purulent_tea": "延长一个效果30秒",
+  "farmersdelight.tooltip.rose_hip_tea": "治疗一颗心",
+  "farmersdelight.tooltip.strong_rose_hip_tea": "治疗两颗心",
+  "farmersdelight.tooltip.strong_melon_juice": "瞬间治疗",
+
+  "farmersrespite.container.kettle": "水壶",
+  "farmersrespite.container.kettle.served_on": "盛装于%s",
+  "farmersrespite.container.kettle.not_heated": "需要下部热源",
+  "farmersrespite.container.kettle.heated": "已加热",
+  "farmersrespite.container.kettle.no_water": "没有饮品",
+  "farmersrespite.container.kettle.has_single_water": "含有 1 份饮品",
+  "farmersrespite.container.kettle.has_many_water": "含有 %s 份饮品",
+
+  "farmersrespite.tooltip.kettle.empty": "空",
+  "farmersrespite.tooltip.kettle.single_serving": "可装 1 份",
+  "farmersrespite.tooltip.kettle.many_servings": "可装 %s 份",
```

```suggestion
  "farmersrespite.tooltip.kettle.single_serving": "可装1份：",
  "farmersrespite.tooltip.kettle.many_servings": "可装%s份：",
```
  "farmersrespite.tooltip.kettle.single_serving": "Holds 1 serving of:",
  "farmersrespite.tooltip.kettle.many_servings": "Holds %s servings of:",